src/iam.ts

```diff
@@ -1084,11 +1084,29 @@ export class IAM extends IAMBase {
    * @returns organization with all nested subOrgs
    *
    */
-  getOrgHierarchy({ namespace }: { namespace: string }) {
+  async getOrgHierarchy({ namespace }: { namespace: string }) {
     if (!this._cacheClient) {
       throw new CacheClientNotProvidedError();
     }
-    return this._cacheClient.getOrgHierarchy({ namespace });
+    const org = await this._cacheClient.getOrgHierarchy({ namespace });
+    const { subOrgs, apps, roles } = org;
+    if (subOrgs?.length && subOrgs.length > 0) {
+      org.subOrgs = await Promise.all(subOrgs.map(
+        async (o) => ({ ...o, isOwned: await this.isOwner({ domain: o.namespace }) })
```

Comment:
Is there a way to prevent calling extra backend service? **iam-client-lib** is still part of frontend.

Reply:
Right. I just wanted to reuse already overridden `isOwner`, but to avoid hops also need to override `getOrgHieararchy`